timeline/templatetags/timeline_tags.py

```diff
@@ -70,24 +69,35 @@ def get_plugin_lookup():
 def get_app_icon_html(event, plugin_lookup):
     """Return icon link HTML for app by plugin lookup"""
     url = None
-    url_kwargs = {}
-    if event.project:
-        url_kwargs['project'] = event.project.sodar_uuid
     title = event.app
     icon = ICON_UNKNOWN_APP  # Default in case the plugin is not found
 
     if event.app == 'projectroles':
         if event.project:
-            url = reverse('projectroles:detail', kwargs=url_kwargs)
+            url = reverse(
+                'projectroles:detail',
+                kwargs={'project': event.project.sodar_uuid},
+            )
         title = 'Projectroles'
         icon = ICON_PROJECTROLES
     else:
         plugin_name = event.plugin if event.plugin else event.app
         if plugin_name in plugin_lookup.keys():
+            url_kwargs = {}
+            if event.project:
```

Comment:
Here, instead of `event.project` we just have to check for the app type. As I wrote in teams, you can do this as follows: `isinstance(your_plugin_here, ProjectAppPluginPoint)`

Aslo, make sure your tests catch this.

Reply:
Implemented this